src/App.vue

```diff
@@ -110,21 +61,6 @@
       />
     </v-alert>
 
-    <div class="app-body">
-      <!-- only show pages while signing in or once the data is loaded -->
-      <main v-if="isSigninRoute || dataLoaded">
-        <Breadcrumb :breadcrumbs="breadcrumbs" />
-        <EntityInfo
-          v-if="!isAmalgamationSelectionRoute"
-          @confirmDissolution="confirmDissolutionDialog = true"
-          @notInGoodStanding="nigsMessage = $event; notInGoodStandingDialog = true"
-          @downloadBusinessSummary="downloadBusinessSummary()"
-          @viewAddDigitalCredentials="viewAddDigitalCredentials()"
-        />
-        <router-view />
```

Comment:
Since App.vue is the main page/container, don't we still need router-view to display the right filing page/view?

Reply:
Reverted.